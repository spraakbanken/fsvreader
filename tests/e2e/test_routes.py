import pytest
from fastapi import status
from httpx import AsyncClient


@pytest.mark.asyncio
async def test_index(client: AsyncClient, snapshot) -> None:
    response = await client.get("/")
    assert response.status_code == status.HTTP_200_OK
    assert response.text == snapshot
