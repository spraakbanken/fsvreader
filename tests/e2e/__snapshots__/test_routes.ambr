# serializer version: 1
# name: test_index
  '''
  <!doctype html>
  <html lang="sv">
      <head>
          <title> FsvReader</title>
          <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
          <link
              rel="shortcut icon"
              href="http://testserver.saldo_ws/static/favicon.ico"
          />
          <script
              src="https://unpkg.com/htmx.org@2.0.3"
              integrity="sha384-0895/pl2MU10Hqc6jd4RvrthNlDiE9U1tWmX7WRESftEDRosgxNsQG/Ze9YMRzHq"
              crossorigin="anonymous"
          ></script>
      </head>
      <body>
          
  <div>
      <h1>FSvReader</h1>
      <p>
          Texterna och metainformationen i FSvReader har hämtats från
          <a href="http://project2.sol.lu.se/fornsvenska"
              >Fornsvenska textbanken</a
          >. För utförligare information, inklusive introducerande kommentarer för
          varje text om inskrivningsprinciperna, hänvisar vi till Fornsvenska
          textbanken.
      </p>
  
      <h4>Kategorier</h4>
      <ul>
          <li>
              <a href="http://testserver.saldo_ws/dir/aldre_lagar"
                  >Äldre lagar</a
              >
          </li>
          <li>
              <a href="http://testserver.saldo_ws/dir/aldre_profan"
                  >Äldre övrigt profan prosa</a
              >
          </li>
          <li>
              <a href="http://testserver.saldo_ws/dir/aldre_religios"
                  >Äldre religiös prosa</a
              >
          </li>
      </ul>
  </div>
  
      </body>
  </html>
  '''
# ---
